hack/jenkins/build.sh

```diff
@@ -47,7 +47,9 @@ docker images -q | xargs -r docker rmi
 # Build
 go run ./hack/e2e.go -v --build
 
-# Push to GCS
-./build/push-ci-build.sh
+[[ ${KUBE_SKIP_PUSH_GCS:-} =~ ^[yY]$ ]] || {
```

Comment:
I was copying KUBE_SKIP_CONFIRMATIONS, KUBE_GCS_UPLOAD_RELEASE etc, which seem to be common in the build scripts (e.g. build/common.sh)

Happy to switch to =="true" if you'd prefer